### PR TITLE
doc(FEC-12795): add readme

### PR DESCRIPTION
**Pull request by giladna**
### Description of the Changes

FEC-12795

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,72 @@
...
+        accountCode: 'YOUR_ACCOUNT', // Replace with your YOUBORA account.
+        application: 'KLTR test',
+        responseTimeoutSec: 15,
+        followRedirects: true,
+        optionalParams: {
+          protocol: 'hls',
+          extended: true,
+          originCode: 'default',
+          live: false,
+          //nva: '',
```
Comment (SivanA-Kaltura) [inline README.md:62]: why is it commented out ?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,72 @@
...
+
+
+### Registering to Events
+
+TBD
+
+
+### Plugin Config
+
+* Make sure your player id is confiured to bundle the playkit-smartswitch: "playkit-smartswitch":"{latest}"
```
Comment (SivanA-Kaltura) [inline README.md:30]:
```suggestion
* Make sure your player id is configured to bundle the playkit-smartswitch: "playkit-smartswitch":"{latest}"
```

File: README.md [hunk 3]
```diff
@@ -1,3 +1,72 @@
...
+
+
+### Plugin Flow 
+
+The `Kaltura Player` triggers an API call towards the Youbora `smart-switch` service with the given playback URL,
+the API will return an ordered list of CDNs. Then the Kaltura Player `smart-switch` plugin will pick the first CDN and will use that CDN new playback url for the playback.
+
+### Error Handling
+
+SmartSwitch errors are handled internally be the plugin and playback will start with the original playback url
```
Comment (SivanA-Kaltura) [inline README.md:20]:
```suggestion
SmartSwitch errors are handled internally by the plugin and playback will start with the original playback url 
```

Comment (SivanA-Kaltura) [reply README.md:20]: wdym by original ?

File: README.md (hunk 1 above)
Comment (SivanA-Kaltura) [reply README.md:62]: do these parameters need an explanation or do we assume that if someone uses the plugin he's familiar with them ?

**Review by SivanA-Kaltura — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (giladna) [reply README.md:62]: i do not have to original document 

but I have example it is there comment

just ti have it as referance

Comment (giladna) [reply README.md:62]: @SivanA-Kaltura ?

File: README.md [hunk 4]
```diff
@@ -1,3 +1,74 @@
...
+the API will return an ordered list of CDNs. Then the Kaltura Player `smart-switch` plugin will pick the first CDN and will use that CDN new playback url for the playback.
+
+### Error Handling
+
+SmartSwitch errors are handled internally by the plugin and playback will start with the original playback url 
+
+
+### Registering to Events
+
+TBD
```
Comment (SivanA-Kaltura) [inline README.md:25]: ?

**Review by SivanA-Kaltura — CHANGES_REQUESTED** [review]

**Review by SivanA-Kaltura — APPROVED** [review]

**Pull request merged**